daemon/container.go

```diff
@@ -201,8 +201,11 @@ func (container *Container) LogEvent(action string) {
 //       symlinking to a different path) between using this method and using the
 //       path. See symlink.FollowSymlinkInScope for more details.
 func (container *Container) GetResourcePath(path string) (string, error) {
-	cleanPath := filepath.Join("/", path)
-	return symlink.FollowSymlinkInScope(filepath.Join(container.basefs, cleanPath), container.basefs)
+	// IMPORTANT - These are paths on the OS where the daemon is running, hence
+	// any filepath operations must be done in an OS agnostic way.
+	cleanPath := filepath.Join(string(os.PathSeparator), path)
```

Comment:
Another great question. At this time we are not supporting drive-letters. (And if/when we support volumes, they will be mounted on a path as opposed to another drive letter). Hopefully we can avoid drive letters all together as it will make cross-platform slashes a LOT simpler to maintain. Hence we can assume here everything is relative to the root of the location on the host filesystem, no drive letter required. Make sense?